config/command.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const Joi = require('joi');
+const Regex = require('./regex');
+const CommandFormat = require('./commandFormat');
+
+const COMMAND_NAMESPACE = Joi
+    .string()
+    .regex(Regex.COMMAND_NAMESPACE)
+    .max(64)
+    .description('Namespace of the Command')
+    .example('chefdk');
+
+const COMMAND_NAME = Joi
+    .string()
+    .regex(Regex.COMMAND_NAME)
+    .max(64)
+    .description('Name of the Command')
+    .example('knife');
+
+const COMMAND_TAG_NAME = Joi
+    .string()
+    .regex(Regex.COMMAND_TAG_NAME)
+    .max(30)
+    .description('Name of the Command Tag')
+    .example('latest');
+
+const COMMAND_VERSION = Joi
+    .string()
+    .regex(Regex.VERSION)
+    .max(16)
+    .description('Version of the Command')
+    .example('1.2');
+
+const COMMAND_EXACT_VERSION = Joi
+    .string()
+    .regex(Regex.EXACT_VERSION)
+    .max(16)
+    .description('Exact version of the Command')
+    .example('1.2.3');
+
+const COMMAND_DESCRIPTION = Joi
+    .string()
+    .max(256)
+    .description('Description of the Command')
+    .example('Installs npm modules');
+
+const COMMAND_MAINTAINER = Joi
+    .string()
+    .email()
+    .max(64)
+    .description('Maintainer of the Command')
+    .example('foo@bar.com');
+
+const COMMAND_FORMAT = Joi
+    .string().valid([
+        'habitat',
+        'docker',
+        'binary'
+    ])
```

Comment:
need `max()` for mysql?
https://github.com/screwdriver-cd/data-schema/pull/102

Reply:
I got it.  I added a length limitation to the  `command.format` property.
Thanks.